### ISSUE TITLE
[A11Y] Modal missing attributes to convey context

### ISSUE THREAD

**Issue by PhilibertDugas** [paypal/paypal-checkout#787]
The PayPal overlay modal is missing a `role`, `aria-modal`, and an `aria-label` attribute. These attributes are used to indicate the presence of a "modal" element which assistive technology users require to help describe the purpose of the current context.

**HTML:**
```html
<div id="xcomponent-paypal-checkout-a1bab52533" class="paypal-checkout-context-popup paypal-checkout-overlay paypal-checkout-background-color-black paypal-checkout-logo-color-white">
  <!-- … -->
</div>

```
## Steps to reproduce

1. Go to https://www.chubbiesshorts.com/ with any supported browser
2. Add some items to the cart
3. Proceed to the Cart
4. Activate the PayPal control
5. `Cmd` + `~` back to the main window
6. With an active screen reader, listen to how the current context is described

### Results

**Expected:**
- Modal to have accompanying attributes to convey context.

**Actual:**
- Modal is missing attributes.

## Recommendation

- Add the `role` attribute and set its value to `dialog`.
- Add the `aria-modal` attribute and set its value to `true`.
- Add the `aria-label` attribute and set its value to describe the purpose of the modal context.

**HTML:**
```html
<div id="xcomponent-paypal-checkout-a1bab52533" class="paypal-checkout-context-popup paypal-checkout-overlay paypal-checkout-background-color-black paypal-checkout-logo-color-white" role="dialog" aria-modal="true" aria-label="PayPal Payments">
  <!-- … -->
</div>

```
## Additional information

- **WCAG Principle:** Robust
- **WCAG SC:** 4.1.2 Name, Role, Value
- **Severity:** High

## Screenshots
<img width="941" alt="screen shot 2018-03-05 at 2 41 22 pm" src="https://user-images.githubusercontent.com/1392632/36996726-7d64d02c-2085-11e8-84c3-224889dce5a6.png">

**Comment by bluepnume** [comment]
@vishakha94 can you take this one?

**Comment by mcntsh** [comment]
This change will be out in today's release. Thanks!
